packages/runtime-web/src/api/device/clipboard/helper.ts

```diff
@@ -0,0 +1,66 @@
+import select from 'select'
+
+/**
+ * Creates a fake textarea element with a value.
+ * @param {String} value
+ * @return {HTMLElement}
+ */
+
+function createFakeElement(value) {
+  const isRTL = document.documentElement.getAttribute('dir') === 'rtl'
+  const fakeElement = document.createElement('textarea')
+  // Prevent zooming on iOS
+  fakeElement.style.fontSize = '12pt'
+  // Reset box model
+  fakeElement.style.border = '0'
+  fakeElement.style.padding = '0'
+  fakeElement.style.margin = '0'
+  // Move element out of screen horizontally
+  fakeElement.style.position = 'absolute'
+  fakeElement.style[isRTL ? 'right' : 'left'] = '-9999px'
+  // Move element to the same position vertically
+  const yPosition = window.pageYOffset || document.documentElement.scrollTop
+  fakeElement.style.top = `${yPosition}px`
+
+  fakeElement.setAttribute('readonly', '')
+  fakeElement.value = value
+
+  return fakeElement
+}
+
+/**
+ * Executes a given operation type.
+ * @param {String} type
+ * @return {Boolean}
+ */
+function command(type) {
+  try {
+    return document.execCommand(type)
+  } catch (err) {
+    return err
+  }
+}
+
+const NOT_SUPPORT_COMMAND = 'your browser not support query command!'
```

Comment:
your browser not support query command! => your browser doesn't support query command!